tests/sentry/incidents/endpoints/test_organization_alert_rule_available_action_index.py

```diff
@@ -11,15 +11,23 @@ def setUp(self):
         super(OrganizationAlertRuleAvailableActionIndexEndpointTest, self).setUp()
         self.login_as(self.user)
 
-    def create_integration_response(self, type, integration=None):
-        return {
+    def create_integration_response(self, type, integration=None, sentry_app=None):
```

Comment:
can we not test with actually using `build_action_response`?

Reply:
I guess this is a helper function, I personally rather have explicit checks. but anyway, do we even have any tests for pagerduty? where is `inputType` now?

Reply:
I think you're right about `build_action_response` as long as I have separate tests for it.

I don't like to have UI logic in the API. Instead of passing `select` or `text`, I infer the input on the fronted based on the presence of the `options` array.